fortinet-fortirecon-easm/info.json

```diff
@@ -3713,6 +3720,261 @@
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "output_schema": {
+        "message": ""
+      },
+      "parameters": [
+        {
+          "name": "asset_id",
+          "title": "Asset ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "description": "Specify the archived asset ID to update the status to active.",
+          "tooltip": "Specify the archived asset ID to update the status to active."
+        }
+      ]
+    },
+    {
+      "operation": "update_archived_issue",
+      "title": "Update Archived Issue",
+      "annotation": "update_archived_issue",
+      "description": "Updates the status of the archived security issue record to active.",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "output_schema": {
+        "message": ""
+      },
+      "parameters": [
+        {
+          "name": "issue_id",
+          "title": "Issue ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "description": "Specify the archived issue ID to update the status to active.",
+          "tooltip": "Specify the archived issue ID to update the status to active."
+        }
+      ]
+    },
+    {
+      "operation": "update_issue_status",
+      "title": "Update Issue Status",
+      "annotation": "update_issue_status",
+      "description": "Updates the status of the Security issue record.\n If the given issue status is set to false_positive or risk_accepted then issue will be visible in the archived issues.",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "output_schema": {
+        "message": ""
+      },
+      "parameters": [
+        {
+          "name": "issue_id",
+          "title": "Issue ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "description": "Specify the security issue ID to update the status.",
+          "tooltip": "Specify the security issue ID to update the status."
+        },
+        {
+          "title": "Status",
+          "name": "status",
+          "type": "select",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "Specify the status to update the security issue.",
+          "description": "Specify the status to update the security issue. You can choose from ACTIVE, RESOLVED, FALSE_POSITIVE, RISK_ACCEPTED.",
+          "options": [
+            "ACTIVE",
+            "RESOLVED",
+            "FALSE_POSITIVE",
+            "RISK_ACCEPTED"
+          ]
+        }
+      ]
+    },
+    {
+      "operation": "update_asn_asset_status_to_false_positive",
```

Comment:
Only one action can be implemented `Update Asset To False Positive` or `Mark Asset As False Positive`  with select parameter `Asset Type` with option `ASN`, `IP Prefix`, `Domain `, `Sub-Domain`.

Reply:
I've purposefully kept it separate, in future we might add more update operations.